assets/javascripts/discourse/initializers/apply-tabs.js.es6

```diff
@@ -0,0 +1,116 @@
+import {withPluginApi} from 'discourse/lib/plugin-api';
+import ComposerController from 'discourse/controllers/composer';
+
+function initializeTabs(api) {
+  api.decorateCooked(
+    $elem => {
+      const radTabsElements = $elem.find(`.js-rad-tabs-element`);
+
+      radTabsElements.each(function() {
+        const radTabsElement = $(this);
```

Comment:
@anthonydillon how do you feel about this? Using jQuery. Maybe we could achieve this with vanilla JS? But I think it could be a more architectural decision, in terms of adding dependencies.

Reply:
Yes, I agree, reducing the number of dependencies is important and jQuery is almost wholly covered by vanilla JS now. 

But, I don't see where jQuery is being injected as a dependency on this project. Is this from Discourse itself?

Reply:
not sure, maybe @albertkol can clarify. But good point from Ant, if discourse itself has jQuery, maybe it doesn't matter then, because it's not our dependency.

Reply:
I can take a look and convert stuff to vanilla JS.

`jQuery` is already loaded by Discourse, so it is available to use.

Reply:
Well if jQuery is available and you are not injecting the dependency anywhere, then I don't see a problem in using it.